src/cmdstan-guide/laplace_sample_config.Rmd

```diff
@@ -0,0 +1,100 @@
+# Laplace sampling
+
+The `laplace` method produces a sample from a Laplace approximation
```

Comment:
The `laplace` method produces a sample from a Laplace approximation
->
The `laplace` method produces a sample from a normal approximation

This change make it more clear that the approximate distribution is normal distribution and not Laplace distribution